app/src/main/res/layout/activity_home.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
It could be a simple tag with a `<androidx.constraintlayout.widget.ConstraintLayout [...] />`.